frontend/src/pages/instance/Settings/Security.vue

```diff
@@ -157,7 +157,7 @@ export default {
     mounted () {
         this.checkAccess()
         this.getSettings()
-        if (this.settings.features.httpBearerTokens) {
+        if (this.settings.features.httpBearerTokens && this.team.type.properties.features.teamHttpSecurity) {
```

Comment:
Please add these checks as computed props in the Features mixin `frontend/src/mixins/Features.js:2` and use the mixin instead.